### PR TITLE
Fix up y1 and y2 value on web page

### PR DESCRIPTION
**Pull request by liuqun**
Update y1 and y2 with earlier changes in mnist/model.py

softmax() is needed to get final y1 and y2 from logit1 and logit2.
Following up with commit 4e84b14e959ba112427759f22b7470c42947f140
And using newest re-trained model data at commit b1a45946431bd9afd86e2d23cb0b429bf08860a3

### CODE REVIEW

**Comment by ghego** [comment]
good catch. I'm not sure why the web page didn't work though, because softmax is monotonic and the js code still looks for the maximum value. What did you mean by "didn't work"?

**Comment by liuqun** [comment]
Thanks. The only problem was that web page used to display wrong float point values, though the web server itself always worked well.
